src/main/java/com/artipie/asto/rx/RxStorage.java

```diff
@@ -66,6 +66,15 @@
      */
     Completable save(Key key, Flowable<ByteBuffer> content);
 
+    /**
+     * Moves value from one location to another.
+     *
+     * @param source Source key.
+     * @param destination Destination key.
+     * @return Completion or error signal.
+     */
+    Completable move(Key source, Key destination);
```

Comment:
@olegmoz If you ask me, RxStorage should extend Storage, so we don't have this kind of duplications... but this is a discussion for another Issue.

Reply:
@amihaiemil whole point is that these are different interfaces, RxStorage is not another implementation of Storage, but storage with reactive interface.